src/Server.hs

```diff
@@ -1,40 +1,54 @@
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE DeriveGeneric #-}
 
-module Server (run) where
+module Server where
 
-import Control.Monad.IO.Class (liftIO)
-import Data.Text.Lazy (pack, Text, unpack)
-import Network.Wai.Middleware.RequestLogger (logStdoutDev)
-import System.Random (getStdGen, randomRIO)
+import GHC.Generics
 import Web.Scotty
+import Data.Text.Lazy (pack, Text, unpack)
+import Data.Monoid ((<>))
+import Data.Aeson (FromJSON, ToJSON)
+import Control.Monad.IO.Class (liftIO)
+import System.Random (randomRIO)
 
-run :: IO ()
-run = do
-  scotty 3000 $ do
-    middleware logStdoutDev
-    routes
+data Question = Question { questionId :: Int, questionText :: String } deriving (Show, Generic)
```

Comment:
Any reason why we don't call it `id` instead of `questionId` and `text` instead of `questionText`?